src/components/dialogs/network-modifications/tabular-creation/tabular-creation-utils.ts

```diff
@@ -107,10 +107,39 @@ export const TABULAR_CREATION_FIELDS: TabularCreationFields = {
         { id: PLANNED_OUTAGE_RATE, required: false },
         { id: FORCED_OUTAGE_RATE, required: false },
     ],
+    BATTERY: [
+        { id: EQUIPMENT_ID, required: true },
+        { id: EQUIPMENT_NAME, required: false },
+        { id: VOLTAGE_LEVEL_ID, required: true },
+        { id: BUS_OR_BUSBAR_SECTION_ID, required: true },
+        { id: CONNECTED, required: true },
+        { id: CONNECTION_NAME, required: false },
+        { id: CONNECTION_DIRECTION, required: false },
+        { id: CONNECTION_POSITION, required: false },
+        { id: MIN_P, required: true },
+        { id: MAX_P, required: true },
+        { id: MINIMUM_REACTIVE_POWER, required: false },
+        { id: MAXIMUM_REACTIVE_POWER, required: false },
+        { id: REACTIVE_CAPABILITY_CURVE, required: true },
+        { id: REACTIVE_CAPABILITY_CURVE_P_MIN, required: false },
+        { id: REACTIVE_CAPABILITY_CURVE_Q_MIN_P_MIN, required: false },
+        { id: REACTIVE_CAPABILITY_CURVE_Q_MAX_P_MIN, required: false },
+        { id: REACTIVE_CAPABILITY_CURVE_P_0, required: false },
+        { id: REACTIVE_CAPABILITY_CURVE_Q_MIN_P_0, required: false },
+        { id: REACTIVE_CAPABILITY_CURVE_Q_MAX_P_0, required: false },
+        { id: REACTIVE_CAPABILITY_CURVE_P_MAX, required: false },
+        { id: REACTIVE_CAPABILITY_CURVE_Q_MIN_P_MAX, required: false },
+        { id: REACTIVE_CAPABILITY_CURVE_Q_MAX_P_MAX, required: false },
```

Comment:
Maybe we could mutualize this part into a common array to spread in the config of equipments having reactive capability curve fields in order to please sonar

Reply:
:+1: 
Sonar issue on code duplication fixed